packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -287,6 +288,12 @@ function OrgSelect({
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const val = e.target.value.replace(/\\/g, '\\');
     setValue(val);
+
+    if (open === true && wasOpen !== open) {
+      // Opt out of type-checking as we just want to trigger the onElementClick handler
+      onElementClick(NavElement.OrgNavOrgSelectSearch)(e as any);
```

Comment:
Not a huge fan of this but the alternative would be to dropdown the type of `onElementClick` when this is the outlier.

Reply:
Could we at least type it as `React.Event` or something slightly more specific like that?

Reply:
I tried a few but none worked.

The issue is we type it as `MouseEvent` which is the most exact.

`interface MouseEvent<T = Element, E = NativeMouseEvent> extends UIEvent<T, E> {`

And using UIEvent:
![image](https://user-images.githubusercontent.com/322576/80129953-da5de880-8565-11ea-9b4d-72b8c25e6aec.png)

Reply:
Ahh yeah, you're right. I suppose the primary issue here is the inconsistency of using the API like this (addressed in my other comment).